bash/formatter.bash

```diff
@@ -9,24 +9,55 @@
 
 function Format_Codebase()
 {
+    local bash_formatter_found='TRUE'
+    local python_formatter_found='TRUE'
     if ! hash shfmt &> /dev/null; then
-        Print_Fatal_And_Exit \
-            'Command ' --emph 'shfmt' ' not available, unable to format codebase.' \
-            'Please, install it (https://github.com/mvdan/sh#shfmt) and run the formatting again.'
-    else
+        Print_Error \
+            'Command ' --emph 'shfmt' ' not available.' \
+            'Please install it (https://github.com/mvdan/sh#shfmt).\n'
+        bash_formatter_found='FALSE'
+    fi
+    if ! hash autopep8 &> /dev/null; then
+        Print_Error \
+            'Command ' --emph 'autopep8' ' not available.' \
+            'Please install it (https://pypi.org/project/autopep8).\n'
+        python_formatter_found='FALSE'
+    fi
+    if [[ "${bash_formatter_found}" = 'TRUE' ]] && [[ "${python_formatter_found}" = 'TRUE' ]]; then
```

Comment:
Why did you put this in `&&`? If I have the Bash formatter but miss the Python one or vice-versa I do not get any formatting done. 🤔 Is this what we want? Suppose I have not installed `autopep8` and changed only Bash code. Then I cannot use `./Hybrid-handler format` because of that... I find this a bit rude, isn't it? 😅 

Also, the `HYBRID_python_folder` variable should be added to the `Ensure_That_Given_Variables_Are_Set_And_Not_Empty` call.

Reply:
Well, for each missing formatter you already give a `Print_Error` above, right? So I would say it is enough. 🤔